chainstate/src/detail/mod.rs

```diff
@@ -130,15 +133,15 @@ impl Chainstate {
         chain_config: Arc<ChainConfig>,
         blockchain_storage: blockchain_storage::Store,
         custom_orphan_error_hook: Option<Arc<OrphanErrorHandler>>,
-        custom_time_getter: Option<Arc<TimeGetter>>,
+        custom_time_getter: TimeGetter,
```

Comment:
This can be renamed to just `time_getter`

Reply:
Done